deployment/ansible/roles/model-my-watershed.rwd/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+- stat: path={{ rwd_data_path }}
+  register: rwd_data
+- fail: msg="Data folder does not exist. Did you forget to mount /opt/rwd-data?"
```

Comment:
It seems like this would fail during the AMI creation phase because `/opt/rwd-data` wouldn't exist.

Wouldn't Vagrant throw some kind of failure if one of the defined shares cannot be setup?